src/mscorlib/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -8,12 +8,14 @@ namespace System.Numerics.Hashing
 
     internal static class HashHelpers
     {
+        public static readonly int RandomSeed = Guid.NewGuid().GetHashCode();
```

Comment:
Thanks for catching that. I accidentally copied over the code that used the random seed. Fixed.